static/js/tools/map/util.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Utility functions shared across different components of map explorer.
+ */
+
+import { StatVarInfo, PlaceInfo } from "./context";
+import { StatsVarNode } from "../statvar_menu/util";
+import _ from "lodash";
+
+const USA_STATE_CHILD_TYPES = ["County"];
+const USA_COUNTRY_CHILD_TYPES = ["State", ...USA_STATE_CHILD_TYPES];
+
+const USA_CHILD_PLACE_TYPES = {
+  Country: USA_COUNTRY_CHILD_TYPES,
+  State: USA_STATE_CHILD_TYPES,
+};
+
+const URLParamKeys = {
+  enclosedPlaceType: "pt",
```

Comment:
these should be treated as constants too, so keys should be e.g. ENCLOSING_PLACE_DCID. same for the enclosing URL_PARAM_KEYS

Reply:
done